app/models/custom_field.rb

```diff
@@ -47,6 +47,13 @@ class CustomField < ActiveRecord::Base
 
   DATA_TYPES = %i(string text number range group boolean)
 
+  def self.loan_questions(field_set = nil)
+    field_set &&= "loan_#{field_set}"
+    field_set ||= ['loan_criteria', 'loan_post_analysis']
```

Comment:
I admit I am stumped here. If the data type of field_set is string, then this seems like a way-too-cutsie way to do things. If it's not, then I'm even more stumped. I would consider refactoring for clarity's sake.